researcher_ids/RI_Claims_V1.md

```diff
@@ -4,7 +4,7 @@
 
 **Version**: 0.9.5 (FROZEN RFC)
 
-**Work Stream Name**: Data Use and Researcher Identity
+**Work Stream Name**: Data Use and Researcher Identity (DURI)
 
 **Product Name**: GA4GH Passports
```

Comment:
Consistency: Passport or Passports? In product description it is singular.

Reply:
Updated reference to be GA4GH Passport in the description and the name of the spec to GA4GH Passport (singular).